### PR TITLE
NWD, NWW working

### PR DESCRIPTION
**Pull request by aniolm**
Third part of homework from cpp-fundamentals.

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @aniolm

**Comment by ziobron** [comment]
⏱ 3 XP extra for @aniolm for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,53 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(lhs==0&&rhs==0){
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:4]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+        return 0;
+    } else {
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: If you return variable here, you don't need to use `else if`

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,53 @@
...
+        }
+        if(rhs<0){
+        rhs*=-1;
+        }
+    
+        for(int i=lhs; i>0; i--){
+            if(lhs%i==0){
+                for(int j=rhs; j>0; j--){
+                    if(rhs%j==0){
+                        if(j==i){
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:26]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 4]
```diff
@@ -1,11 +1,53 @@
...
+        return 0;
+    }else{
+        if(lhs<0){
+        lhs*=-1;
+        }
+        if(rhs<0){
+        rhs*=-1;
+        }
+        return (lhs * rhs) / NWD(lhs,rhs);
+    }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:51]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 5]
```diff
@@ -1,11 +1,53 @@
...
+    
+    return 1;
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(lhs==0||rhs==0){
+        return 0;
+    }else{
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:43]: If you return variable here, you don't need to use `else if` or `else`

File: homework/nwd-nnw/nwdNww.hpp [hunk 6]
```diff
@@ -1,11 +1,53 @@
...
+        if(lhs<0){
+        lhs*=-1;
+        }
+        if(rhs<0){
+        rhs*=-1;
+        }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:49]: Add space between operations like '+' '-' '!=' '==' '%' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**